account_fiscal_position_rule/__manifest__.py

```diff
@@ -8,21 +8,26 @@
 
 {
     'name': 'Account Fiscal Position Rule',
-    'version': '10.0.1.0.0',
+    'summary': 'Account Fiscal Position Rule',
     'category': 'Generic Modules/Accounting',
-    'author': "Akretion, Odoo Community Association (OCA)",
+    'version': '12.0.1.0.0',
+    'author': (
+        'Akretion',
+        'Odoo Community Association (OCA)',
+    ),
```

Comment:
what did you changed the syntax ? 
I don't see it as the normal manner to define author. 
Ref : https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L12